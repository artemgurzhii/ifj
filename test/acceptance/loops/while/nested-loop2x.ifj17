dim a as integer
dim res as integer
a = 1
do while a != 1
  do while a != 1
    res = 42
  loop
loop

do while a != 1
  do while a != 1
    res = 42
  loop
loop
