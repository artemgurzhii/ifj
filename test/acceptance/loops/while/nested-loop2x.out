
.IFJcode17
JUMP Scope
LABEL Scope
DEFVAR GF@a
DEFVAR GF@res
MOVE GF@a int@1
LABEL LOOP_1
LABEL LOOP_2
MOVE GF@res int@42
JUMPIFNEQ LOOP_2 GF@a int@1
JUMPIFNEQ LOOP_1 GF@a int@1
LABEL LOOP_3
LABEL LOOP_4
MOVE GF@res int@42
JUMPIFNEQ LOOP_4 GF@a int@1
JUMPIFNEQ LOOP_3 GF@a int@1
